Assets/MixedRealityToolkit.Extensions/Webrtc/Plugins/Win32/x64/webrtc_unity_plugin.dll.meta

```diff
@@ -0,0 +1,128 @@
+fileFormatVersion: 2
+guid: 5b5f8d16de5c2e248bd1ef2a386bbecd
+PluginImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  iconMap: {}
+  executionOrder: {}
+  defineConstraints: []
+  isPreloaded: 0
+  isOverridable: 0
+  isExplicitlyReferenced: 0
+  platformData:
+  - first:
+      '': Any
```

Comment:
we should probably avoid having *.dll.meta files as well. Nuget for unity won't drop dlls into this path/you'll want to include meta files in the nuget packages that you distribute with your binaries if you ever distribute nuget packages.

Reply:
ah ok. my thinking was that this makes it clearer that some dlls need to get placed, but you're right that it comes at a cost of "clutter" that likely won't be viable for folks. I'll cleanup.